pkg/antctl/raw/supportbundle/command_test.go

```diff
@@ -351,17 +403,24 @@ func TestProcessResults(t *testing.T) {
 				defaultFS = afero.NewOsFs()
 			}()
 
-			err := processResults(tt.resultMap, option.dir)
-			if tt.expectedErr != "" {
-				require.ErrorContains(t, err, tt.expectedErr)
-			} else {
-				require.NoError(t, err)
-			}
-			// Both test cases above have failed Nodes, hence this file should always be created/
+			antreaInterface := fakeclientset.NewSimpleClientset(&controllerInfo, agentInfo1, agentInfo2)
+			k8sClient := fake.NewSimpleClientset(controllerPod, pod1, pod2)
+			err := processResults(context.TODO(), antreaInterface, k8sClient, tt.resultMap, option.dir)
+			require.NoError(t, err)
 			b, err := afero.ReadFile(defaultFS, filepath.Join(option.dir, "failed_nodes"))
 			require.NoError(t, err)
 			data := string(b)
 			for node, err := range tt.resultMap {
+				fileName := fmt.Sprintf("agent_%s.tar.gz", node)
+				if node == "" {
+					fileName = "controller.tar.gz"
+				}
+				if err != nil {
+					ok, checkErr := afero.Exists(defaultFS, filepath.Join(option.dir, fileName))
+					require.NoError(t, checkErr)
+					assert.True(t, ok, fmt.Sprintf("expected support bundle file %s not found", fileName))
```

Comment:
you don't need an explicit call to `fmt.Sprintf` here. You can use `assert.Truef` if you want, but if I remember correctly, `assert.Truef` and `assert.True` are actually equivalent.